src/EditorFeatures/CSharpTest/CodeActions/ReplaceMethodWithProperty/ReplaceMethodWithPropertyTests.cs

```diff
@@ -35,7 +35,7 @@ public void TestMethodWithArrowBody()
         {
             Test(
 @"class C { int [||]GetFoo() => 0; }",
-@"class C { int Foo { get; } => 0; }");
+@"class C { int Foo => 0; }");
```

Comment:
Sure!  What do we use the workitems for BTW?  I noticed a bunch of them are just random numbers.  And I don't even know what bug DB they're even referring to...